src/api/app/views/webui2/shared/_editor.html.haml

```diff
@@ -1,6 +1,6 @@
 :ruby
   save ||= {}
-  style ||= { read_only: false }
+  style ||= { read_only: false, big_editor: false }
```

Comment:
Hm, we only have one page where this should be false, `BsRequest#show`. How about turning this default around?

Reply:
I agree. I will change the default.

Reply:
Done. Should I merge all the commits into one?

Reply:
Yeah sounds like a good idea :)